frontend/src/components/borrow/RequestLoan.vue

```diff
@@ -80,13 +81,14 @@ const Component = Vue.extend({
     },
     requestLoan: async function requestLoan() {
       this.loading = true;
-      const daysToSeconds = seconds => seconds * 24 * 60 * 60;
-      const loan: Loan = await API.instance().addNewLoan(
+      const fundraisingEnd: moment.Moment = moment().add(7, 'days');
+      const paybackEnd: moment.Moment = moment().add(7 + this.paybackTime, 'days');
```

Comment:
How about:

`const paybackEnd: moment.Moment = fundraisingEnd.add(this.paybackTime, 'days');`